src/test/java/org/broadinstitute/hellbender/tools/walkers/haplotypecaller/HaplotypeCallerIntegrationTest.java

```diff
@@ -493,6 +493,28 @@ public void testSitesOnlyMode() {
         }
     }
 
+    @Test
+    public void testForceActiveOption() throws Exception {
+        Utils.resetRandomGenerator();
+        final File out = createTempFile("GTStrippedOutput", "vcf");
+        final File assemblyRegionOut = createTempFile("assemblyregions", ".igv");
+        final File expectedAssemblyRegionOut = new File(TEST_FILES_DIR, "expected.testAssemblyRegionWithForceActiveRegions_assemblyregions.igv");
+
+        final String[] args = {
+                "-I", NA12878_20_21_WGS_bam,
+                "-R", b37_reference_20_21,
+                "-L", "20:1-5000",
+                "-O", out.getAbsolutePath(),
+                "-pairHMM", "AVX_LOGLESS_CACHING",
+                "--" + AssemblyRegionWalker.FORCE_ACTIVE_REGIONS_LONG_NAME, "true",
```

Comment:
Can you confirm that this test case fails if you omit the `--force-active` option? Ie., try running it without the `--force-active` option in the command line and see what happens.

Reply:
It fails. It outputs bunch of -1s in "assembly-region-out" output instead of 1s